CMakePresets.json

```diff
@@ -12,6 +12,9 @@
       "hidden": true,
       "binaryDir": "${sourceDir}/out/build/${presetName}",
       "installDir": "${sourceDir}/out/install/${presetName}",
+      "environment": {
+        "DONT_CONFIGURE_QT": "1"
```

Comment:
what happens in an environment where this variable is not set at all? From what I see that means that's the same as DONT_CONFIGURE_QT=0, i.e. it will be **not** _not_ configured.

I am not sure whether that should be the default, and double negatives are a pain. Can this be inverted to `"CONFIGURE_QT": "0"`?

Reply:
Yes you are totally right. Resolved this brainf**k :)